src/hooks/useUnstake.ts

```diff
@@ -27,22 +27,19 @@ const useUnstake = (pid: number) => {
   return { onUnstake: handleUnstake }
 }
 
-const SYRUPIDS = [5, 6, 3, 1, 22, 23, 78]
```

Comment:
Pool 78 is still in [config file](https://github.com/pancakeswap/pancake-frontend/blob/develop/src/config/constants/pools.ts#L518) and doesn't have `emergencyWithdraw` flag defined. Should either remove pool 78 or add flag in there?

Reply:
@Chef-Chungus asked me to not touch the config file because he is working on it. He will add the `emergencyWithdraw` param in an other pr.

Reply:
got it 👍 
PS yeah what was I thinking of removing it, it still has 200k CAKE in it 😅